src/core/process_output.mjs

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the ""License"");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an ""AS IS"" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+'use strict'
+import { ProcessOutput } from 'listr2'
+
+/**
+ * Uses the solo logger to handle process output from Listr2
+ * @class CustomProcessOutput
+ * @augments ProcessOutput
+ */
+export class CustomProcessOutput extends ProcessOutput {
+  /** @param {SoloLogger} logger */
+  constructor (logger) {
+    super()
+    /** @private */
+    this._logger = logger
+  }
+
+  /** @inheritDoc */
+  toStdout (chunk, eol = true) {
```

Comment:
can we do a split on '\n' to turn it into multiple log statements? 
from this:
```
{"label":"","level":"DEBUG","message":"\u001b[32m✔\u001b[39m Initialize\n\u001b[32m✔\u001b[39m Prepare chart values\n\u001b[32m✔\u001b[39m Install 'fullstack-cluster-setup' chart \u001b[32m[17s]\u001b[39m","ms":"+2ms","timestamp":"2024-09-30T14:21:44.573Z","traceId":"84878b72-68f2-4710-911d-94e02cc9a1bc"}
```
which I see using JQ as this:
```
2024-09-30T14:21:44.573Z	✔ Initialize\n✔ Prepare chart values\n✔ Install 'fullstack-cluster-setup' chart [17s]
```
into this:
```
2024-09-30T14:21:44.573Z	✔ Initialize
2024-09-30T14:21:44.573Z	✔ Prepare chart values
2024-09-30T14:21:44.573Z	✔ Install 'fullstack-cluster-setup' chart [17s]
```

Reply:
Fixed